test/browser_tests/page_objects/page_activity-log.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+function ActivityLog() {
+  this.get = function() {
+    browser.get( '/activity-log/' );
+  };
+
+  this.pageTitle = function() { return browser.getTitle(); };
+
+  this.mainTitle = element( by.css( '[data-qa-hook="main-title"]' ) );
+
+  this.mainSummary = element( by.css( '[data-qa-hook="main-summary"]' ) );
+
+  this.searchFilter = element( by.css( '.js-post-filter' ) );
```

Comment:
No, I mean `data-qa-*`. It looks like https://github.com/cfpb/cfgov-refresh/pull/969/files#diff-fb399c36e6cf690fc86d258a0e3e666cR26 is accessing the tbody in the same div. The HTML looks like:

```
<div class="block__sub u-mb0" data-qa-hook="filter">
    …
    <div class="expandable expandable__filters expandable__padded js-post-filter">
        …
    </div>
    <table class="u-w100pct">
         <tbody>
             …
```

So just wondering if this one can't utilize the `data-qa-*` hook like `searchFilterResults` does? Otherwise it seems like `data-qa-hook="filter"` should actually be on the `<tbody>` in the HTML instead.